### PR TITLE
Sp2 us1

### PR DESCRIPTION
**Pull request by donovanrost**

### CODE REVIEW

File: binnacle/lib/models/provider_data.dart [hunk 1]
```diff
@@ -1,18 +1,20 @@
-
+import '../enums.dart';
```
Comment (willnwalker) [inline binnacle/lib/models/provider_data.dart]: I think the name should be more descriptive, even if there's only one kind of enum present in the file. We might use enums elsewhere, something like ProviderTypeEnums would do.

File: binnacle/lib/services/bluetooth_parser.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import 'package:sos/models/compass_model.dart';
+import 'package:sos/models/position_model.dart';
+import 'package:sos/models/wind_model.dart';
+import '../enums.dart';
+
+/*
```
Comment (willnwalker) [inline binnacle/lib/services/bluetooth_parser.dart]: We should add a TODO: update packet format annotation

Comment (CaseyHillers) [reply binnacle/lib/services/bluetooth_parser.dart]: Also, dart style uses /// comments for documentation

File: binnacle/lib/models/provider_data.dart (hunk 1 above)
Comment (donovanrost) [reply binnacle/lib/models/provider_data.dart]: So my thought was just to have a file that just holds all the enums

**Pull request merged**